api/gas/gas.go

```diff
@@ -10,6 +10,12 @@ import (
 	"github.com/renproject/pack"
 )
 
+type TxType uint8
+
+const (
+	ETHTransfer = TxType(0)
+)
```

Comment:
It is an anti-pattern to need to define blockchain specific types/constants in the API packages. See the other comment regarding _not_ estimating gas limits yet.